fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -598,7 +601,16 @@ public String logKey() {
         DELETES("deletes", false),
         /** Total number of mutation operations. */
         MUTATIONS("mutations", false),
-        ;
+        /** JNI Calls in FDB.*/
+        FDB_JNI_CALLS("FDB jni calls",false),
+        /**Bytes read from FDB.*/
+        FDB_BYTES_FETCHED("FDB bytes fetched",false),
+        /** Number of network fetches performed by FDB.*/
+        FDB_RANGE_QUERY_FETCHES("FDB Range query fetches",false),
+        /** Number of Key-values fetched during a range scan.*/
+        FDB_RANGE_QUERY_KEYVALUES_FETCHED("FDB range keyvalues ",false ),
```

Comment:
Maybe:

```
        RANGE_FETCHES("range fetches", false),
        RANGE_KEYVALUES_FETCHED("range key-values", false )
```

Note the `FDB` dropped. The `QUERY` is a bit confusing/misleading since we have a whole class of `QUERY` metrics surrounding actual record queries.  Also a few minor nits:  For consistency try to avoid capitalization (`Range` -> `range), and put a space after the comma between the name and the `false`.